scripts/pmm_kodiak.py

```
@@ -0,0 +1,462 @@
+            connector = self.connectors[self.config.exchange]
+            connector.set_leverage(self.config.trading_pair, self.config.leverage)
+            self.account_config_set = True
+    
+    def create_proposal(self) -> List[PerpetualOrderCandidate]:
+        connector = self.connectors[self.config.exchange]
+        # Use mark price from exchange instead of mid price
+        mark_price = connector.get_price_by_type(self.config.trading_pair, PriceType.MarkPrice)
+        # best_bid_price = connector.get_price_by_type(self.config.trading_pair, PriceType.BestBid)
+        # best_ask_price = connector.get_price_by_type(self.config.trading_pair, PriceType.BestAsk)
+        
+            # Get order book
```

Flake8 (7.3.0)

[error] unexpected indentation (comment)

(E116)